packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -1496,7 +1496,6 @@
 			baseConfigurationReference = 9F29408327A999D200AB150A /* Debug-iOS.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				CLANG_CXX_LANGUAGE_STANDARD = "c++14";
```

Comment:
Can be taken directly from the podspec files, so we better no set it here

Reply:
Did you manually edit this?  Internally at Microsoft we try to move all build settings to .xcconfig files so they're easier to manage. I guess that's another opportunity for an upstream change.